### PR TITLE
Day 13 Ex 01 Chat Using Nine Patch Images

### PR DESCRIPTION
**Pull request by at-thienngnuyen**
#### Related issues (Optional)
Tag #66 

#### Description (Required)
- [x] Ui chat
 - [x] Logic Chat

#### Reference (Optional)

### CODE REVIEW

File: app/src/main/res/layout/fragment_content_select.xml [hunk 1]
```diff
@@ -52,5 +52,33 @@
...
+                android:id="@+id/btnEx11.2"
+                style="@style/ButtonNormalStyle"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                android:layout_weight="1"
+                android:text="@string/button.text.ex11_2" />
+        </LinearLayout>
+
+        <Button
+            android:id="@+id/btnEx13"
```
Comment (at-dev-support-01) [inline app/src/main/res/layout/fragment_content_select.xml]: WARNING: [InconsistentLayout] The id "btnEx13" in layout "fragment_content_select" is missing from the following layout configurations: layout-v21 (present in layout)
 This check ensures that a layout resource which is defined in multiple resource folders, specifies the same set of widgets.

This finds cases where you have accidentally forgotten to add a widget to all variations of the layout, which could result in a runtime crash for some resource configurations when a `findViewById()` fails.

There *are* cases where this is intentional. For example, you may have a dedicated large tablet layout which adds some extra widgets that are not present in the phone version of the layout. As long as the code accessing the layout resource is careful to handle this properly, it is valid. In that case, you can suppress this lint check for the given extra or missing views, or the whole layout

**Comment by at-vinhhuynh** [comment]
LGTM

**Comment by at-tienhoang** [comment]
@at-thienngnuyen Please change the description

**Comment by at-thienngnuyen** [comment]
@at-tienhoang I changed and fixed conflict. Please check.
Thanks

**Comment by at-tienhoang** [comment]
@at-thienngnuyen Please resolve conflict

**Comment by at-tienhoang** [comment]
LGTM! thanks

**Pull request merged**